src/views/studio/CreateEditChannelView/CreateEditChannelView.tsx

```diff
@@ -179,33 +186,40 @@ const CreateEditChannelView: React.FC<CreateEditChannelViewProps> = ({ newChanne
     setTransactionStatus(ExtrinsicStatus.ProcessingAssets)
 
     const assets: ChannelAssets = {}
+    let avatarContentId = ''
+    let coverContentId = ''
+
     if (dirtyFields.avatar) {
       if (data.avatar.blob && avatarHashPromise) {
-        assets.avatar = {
+        ;[assets.avatar, avatarContentId] = joystream.createFileAsset({
```

Comment:
It's not only not a mistake, it's actually required here. This is one of the exceptions in which you need to explicitly add a semicolon even though we are not usually using them. This is because the line is starting with `[`. I've changed it now so we don't need that

Reply:
Actually, thinking about it now, in this exact setup the semicolon may not be required because it's a first line after an opening brace, so would likely be parsed properly but Prettier will insert a semicolon at the start of any line that starts with a `[` or `(`